bridges/relays/client-substrate/src/test_chain.rs

```diff
@@ -56,6 +56,7 @@ impl bp_runtime::Chain for TestChain {
 impl Chain for TestChain {
 	const NAME: &'static str = "Test";
 	const BEST_FINALIZED_HEADER_ID_METHOD: &'static str = "TestMethod";
+	const FREE_HEADERS_INTERVAL_METHOD: &'static str = "TestMethod";
```

Comment:
is it ok to have the same name for both these different methods?

Reply:
Yeah, it's just for tests and we do not have a mocked clients, which are able to respond to RPC calls => it is not tested